jupyterlab_hub/src/plugin.ts

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+import {
+  Menu
+} from 'phosphor/lib/ui/menu';
+
+import {
+  getBaseUrl, getConfigOption
+} from '@jupyterlab/services/lib/utils';
```

Comment:
We're discouraging importing from fully qualified paths in `@jupyterlab`.  Import the utils namespace and access these functions as `utils.getBaseUrl`.